cgt_calc/parsers/__init__.py

```diff
@@ -127,3 +155,17 @@ def read_initial_prices(
             initial_prices[date_index] = {}
         initial_prices[date_index][entry.symbol] = entry.price
     return initial_prices
+
+
+def read_isin_translation_file(
+    isin_translation_file: Traversable | Path,
+) -> dict[str, set[str]]:
+    """Read ISIN translation data to tickers from the input path."""
+    with isin_translation_file.open(encoding="utf-8") as csv_file:
+        lines = list(csv.reader(csv_file))
+        lines = lines[1:]
```

Comment:
let's be more strict, we don't want to be reading the wrong data/file.
1. check that the header matches as safety check
2. perform the validation on the result, each file should be valid by itself.